File: DnsTube.Core/Models/DnsEntry.cs
namespace DnsTube.Core.Models
{
	public class DnsEntry
	{
		public bool? UpdateCloudflare { get; set; }
		public string? DnsName { get; set; }
		public string? Type { get; set; }
		public string? Address { get; set; }
		public int? TTL { get; set; }
		public bool? Proxied { get; set; }
		public string? ZoneName { get; set; }
		public string? NetworkAdapterName { get; set; }
	}
}

File: DnsTube.Service/ClientApp/src/model/DnsEntry.ts
export class DnsEntry {
	public updateCloudflare: boolean | undefined;
	public dnsName: string | undefined;
	public type: string | undefined;
	public address: string | undefined;
	public ttl: number | undefined;
	public proxied: boolean | undefined;
	public zoneName: string | undefined;
}